### DISCUSSION TITLE
How to change application accent color before any window show up?

### DISCUSSION THREAD

**Discussion by SulfuricAcidH2SO4** [lepoco/wpfui#981] [Q&A]
Basically as the title says.

I tried running in the startup of my App.xaml.cs 

`ApplicationAccentColorManager.Apply(Colors.Red, ApplicationTheme.Dark);`

but it doesnt seem to work, it keeps reverting back to windows default accent color. Not sure when should I call it and why it doesn't change in the slightest....

**Comment by GalaxyGaming2000** [comment]
If i am correct you do this in app.xaml.cs:

```csharp
using Wpf.Ui.Appearance;

namespace YOURNAMESPACEHERE
{
    public partial class App : Application
    {
        protected override void OnStartup(StartupEventArgs e)
        {
            base.OnStartup(e);
            ApplicationAccentColorManager.Apply(Colors.Red);
        }
    }
}
```

**Comment by SamKr** [comment]
This appears to be a bug which I'm hoping will be resolved soon: https://github.com/lepoco/wpfui/issues/965
But haven't heard back from @pomianowski about it unfortunately.

Can't be fixed by @GalaxyGaming2000's solution either.
